Source/WebKit/NetworkProcess/NetworkProcess.cpp

```diff
@@ -339,7 +339,7 @@ void NetworkProcess::initializeNetworkProcess(NetworkProcessCreationParameters&&
 #endif
 
     for (auto [processIdentifier, domain] : parameters.allowedFirstPartiesForCookies) {
-        if (auto* connection = webProcessConnection(processIdentifier))
+        if (RefPtr connection = webProcessConnection(processIdentifier))
```

Comment:
Should `webProcessConnection` return a `RefPtr`?

Reply:
We don't do it in the case we want to write code like this where foo() and bar() are trivial functions.
webProcessConnection(processIdentifier)->foo()->bar();